dotCMS/src/main/java/com/dotcms/cms/login/LoginServiceAPIFactory.java

```diff
@@ -382,6 +385,7 @@ private void processJsonWebToken(final HttpServletRequest req,
 
 
         @Override
+        @CloseDBIfOpened
         public boolean doLogin(final LoginForm form,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "LoginForm"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)